src/fronts/business-today/variantB/components/LiveCollection.tsx

```diff
@@ -28,11 +28,7 @@ export const LiveCollection: React.FC<{
             />
             {content.map((story, index) => (
                 <>
-                    <OverlayCard
-                        content={story}
-                        salt={salt}
-                        isLive={!!story.card.isLive}
-                    />
+                    <OverlayCard content={story} salt={salt} isLive />
```

Comment:
We forgot to make the `isLive` prop dynamic again. Needs to be `isLive={story.card.isLive}`.